docs/installing.rst

```diff
@@ -2,8 +2,71 @@
 Installing the Atomic Developer Bundle
 ======================================
 
+------------------------------------
+1. Install a Virtualization Provider
+------------------------------------
+
+Two virtualization providers have been tested with the ADB.
+
+* Microsoft Windows and Mac OS X
+
+  The suggested virtualization provider is `VirtualBox <https://www.virtualbox.org/>`_.  Installation instructions are available `online <https://www.virtualbox.org/manual/UserManual.html>`_.  While the latest stable shipping release should work, the majority of testing has been done with version 5.0.0 on Mac OS X and **XXX** on Microsoft Windows.
+
+* Fedora
+
+  Two different virtualization providers are supported on Linux, `VirtualBox <https://www.virtualbox.org/>`_ and `libvirt <http://libvirt.org/>`_.  The choice as to which to use should be driven by your preferences and environmental concerns and is outside of the scope of this document.  Both will work equally well in their default configuration.  You may wish to read the section on file synchronization when making this decision.
+
+  * VirtualBox Installation instructions are available `online at the VirtualBox website <https://www.virtualbox.org/manual/UserManual.html>`_.
+
+    A summary of the installation is listed below:
+
+    ::
+
+      $ dnf install dkms
+      $ curl -O http://download.virtualbox.org/virtualbox/rpm/fedora/virtualbox.repo
+      $ mv virtualbox.repo /etc/yum.repos.d/
+      $ dnf install VirtualBox-4.3
+      $ sudo /etc/init.d/vboxdrv setup
+    
+    While the latest stable shipping release should work, the majority of testing has been done with version 4.3.30.
+
```

Comment:
For today, I'd like to leave it here.  Mostly because the current Vagrant for CentOS doesn't support the latest VirtualBox.  Once things are flowing more smoothly, we can fix that.

If I wind up refactoring this document into target based installation pages, I'll include the note only where needed and put the tested versions somewhere else.